src/_nebari/upgrade.py

```diff
@@ -39,6 +45,20 @@
 
 
 def do_upgrade(config_filename, attempt_fixes=False):
+    """
+    Perform an upgrade of the Nebari configuration file.
+
+    This function loads the YAML configuration file, checks for deprecated keys,
+    validates the current version, and if necessary, upgrades the configuration
+    to the latest version of Nebari.
+
+    Parameters:
```

Comment:
Hi @arjxn-py, in Nebari we use the Google style guide for docstrings (https://google.github.io/styleguide/pyguide.html#381-docstrings). That specifies using `Args` instead of `Parameters`

Reply:
I've switched to specify it as `Args` instead of `Parameters` in [`52ea69e`](https://github.com/nebari-dev/nebari/pull/2512/commits/52ea69e511a1ed309f03931edf857dd0914483dc)